examples/er-rest-example/resources/res-b1-sep-b2.c

```diff
@@ -67,9 +67,9 @@ res_post_handler(void *request, void *response, uint8_t *buffer, uint16_t prefer
     coap_separate_accept(request, &request_metadata);
 
     /* Need Time for calculation now */
-    uint32_t i;
+    int i;
```

Comment:
The fix was on the type size rather than signedness. Now switching to unsigned, thanks